File: solutions/fizzbuzz/fizzbuzz.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    // Loop from 1 to 100
    for(int i=1;i<=100;i++) {
        // Check if the number is divisible by 3
        if(i%3==0 && i%5==0) cout << "FizzBuzz" << endl;
        // Check if the number is divisible by 5
        else if(i%3==0) cout << "Fizz" << endl;
        // This condition will never be true because it is already checked in the first condition
        else if(i%5==0) cout << "Buzz" << endl;
        // If none of the above conditions are true, print the number
        else cout<<i<<endl;
    }
    return 0;
}

File: solutions/fizzbuzz/fizzbuzz.js
//code solution by Satyam Lal

// Loop from 1 to 100
for (let i = 1; i <= 100; i++) {
    // Check if the number is divisible by both 3 and 5
    if (i % 3 === 0 && i % 5 === 0) {
        console.log("FizzBuzz");
    }
    // Check if the number is divisible by 3
    else if (i % 3 === 0) {
        console.log("Fizz");
    }
    // Check if the number is divisible by 5
    else if (i % 5 === 0) {
        console.log("Buzz");
    }
    // If none of the above conditions are true, print the number
    else {
        console.log(i);
    }
}
